src/utils/domains.ts

```diff
@@ -1,10 +1,15 @@
 import { Colony } from '~types';
 import { notNull } from './arrays';
 
-export const findDomain = (domainId?: string | null, colony?: Colony) => {
```

Comment:
We were looking up the domain using its database ID, whereas actions data references the native domain IDs